### PR TITLE
[linear_equations] Translation

### PR DESCRIPTION
**Pull request by longye-tian**
Hi all @mmcky @HumphreyYang @SylviaZhaooo ,

I have translated the lecture linear_equations.md and put the translated version in the lecture folder.

Best,
Longye

### CODE REVIEW

**Comment by mmcky** [comment]
@longye-tian this build is failing due to 

```
/home/runner/work/lecture-intro.zh-cn/lecture-intro.zh-cn/lectures/linear_equations.md:806: WARNING: equation not found: la_se2
```

are you able to look this up to see what might be the issue?

**Comment by mmcky** [comment]
@longye-tian I gave this PR a go to see if some white space suggestions seem to fix the equation issue.

**Comment by mmcky** [comment]
Equations are definitely buggy in the translations. 

The build is now reporting

```
/home/runner/work/lecture-intro.zh-cn/lecture-intro.zh-cn/lectures/linear_equations.md:858: WARNING: equation not found: la_se2
/home/runner/work/lecture-intro.zh-cn/lecture-intro.zh-cn/lectures/linear_equations.md:902: WARNING: equation not found: two_eq_demand_mat
/home/runner/work/lecture-intro.zh-cn/lecture-intro.zh-cn/lectures/linear_equations.md:902: WARNING: equation not found: two_eq_supply_mat
```

I wonder if its a mix of `math` directives and using `$$` math syntax. 

I will do some more tests tomorrow unless anyone else has good ideas.

**Comment by longye-tian** [comment]
> Equations are definitely buggy in the translations.
> 
> The build is now reporting
> 
> ```
> /home/runner/work/lecture-intro.zh-cn/lecture-intro.zh-cn/lectures/linear_equations.md:858: WARNING: equation not found: la_se2
> /home/runner/work/lecture-intro.zh-cn/lecture-intro.zh-cn/lectures/linear_equations.md:902: WARNING: equation not found: two_eq_demand_mat
> /home/runner/work/lecture-intro.zh-cn/lecture-intro.zh-cn/lectures/linear_equations.md:902: WARNING: equation not found: two_eq_supply_mat
> ```
> 
> I wonder if its a mix of `math` directives and using `$$` math syntax.
> 
> I will do some more tests tomorrow unless anyone else has good ideas.

Hi Matt @mmcky ,

Just add more empty lines.
Now I think it works.

Best,
Longye

File: lectures/linear_equations.md [hunk 1]
```diff
@@ -668,25 +734,32 @@ $$
 ```{math}
 :label: two_matrix
 
-p = (C - D)^{-1} h.
+    p = (C - D)^{-1} h.
```
Comment (mmcky) [inline lectures/linear_equations.md:737]: @longye-tian these indentations shouldn't be needed. Do you add them for style?

**Review by mmcky — APPROVED** [review]
thanks @longye-tian -- it seems there needs to be white space around the `:label:` keys

File: lectures/linear_equations.md (hunk 1 above)
Comment (longye-tian) [reply lectures/linear_equations.md:737]: I'll change it back. I think I put too much space after the : label:

**Pull request merged**